### ISSUE TITLE
When a tiddler is manually added to the list field of a kanban board column tiddler, the kanban history of the card tiddler is not updated to show when it was added

### ISSUE THREAD

**Issue by AshRubigo** [BurningTreeC/tiddlywiki-muuri-kanban#4]
**Environment:**
Version: Main github branch.
Browser: Firefox 90.0.2.
TiddlyWiki: 5.1.23, 5.2.0-prerelease.

**Steps to reproduce:**

1. Create a kanban board tiddler with a column.
2. Create a tiddler.
1. Manually add the tiddler to the list field of the column tiddler.

**Expected result:**

The card tiddler would have its kanban-history field updated to show when it was added to the column.

**Comment by BurningTreeC** [comment]
Hi @AshRubigo , I think I can't do anything about that issue. There needs to be a user-interaction using a button or a droppable field so that I can populate the kanban-history field

**Comment by AshRubigo** [comment]
I thought that might be the case. One solution would be to use the 'Confirm changes to this tiddler' button. When that is pressed, check all the tiddlers in the list field to see if they have a kanban-history field, and if not create it and add an item.

**Comment by BurningTreeC** [comment]
Yes that would be a solution, but that would force me to edit a tiddlywiki core tiddler, which is something I want to avoid
